### ISSUE TITLE
One of the two websocket reconnected automatically

### ISSUE THREAD

**Issue by emmtte** [binance/binance-connector-node#40]
### Issue subject
One of the two websocket reconnected automatically

### Expected behaviour
close the two websocket

### Actual behaviour
````
wss://stream.binance.com:9443/ws/bnbusdt@kline_1m
listen to event: open
listen to event: close
listen to event: message
wss://stream.binance.com:9443/ws/btcusdt@kline_1m
listen to event: open
listen to event: close
listen to event: message
open
open
{"e":"kline","E":1630049890241,"s":"BTCUSDT","k":...............
{"e":"kline","E":1630049891352,"s":"BNBUSDT","k":...............
closed
closed
Reconnect to the server.
listen to event: open
listen to event: close
listen to event: message
open
{"e":"kline","E":1630049901181,"s":"BTCUSDT","k":..................
````

### Steps to reproduce
modified example from https://github.com/binance/binance-connector-node/blob/master/examples/websocket/spot/klineWS.js

````
const { Console } = require('console')
const fs = require('fs')
const { Spot } = require('@binance/connector')

const output = fs.createWriteStream('./stdout.log')
const errorOutput = fs.createWriteStream('./stderr.log')

const logger = new Console({
   stdout: output,
   stderr: errorOutput
 })

const client = new Spot('', '', {
logger
})

const callbacks = {
open: () => client.logger.debug('open'),
close: () => client.logger.debug('closed'),
message: data => client.logger.log(data)
  }

const wsRef   = client.klineWS('bnbusdt', '1m', callbacks) 
const wsRef1 = client.klineWS('btcusdt', '1m', callbacks)

setTimeout(() => client.unsubscribe(wsRef), 5000)
setTimeout(() => client.unsubscribe(wsRef1), 5000)
// check the output file
````

### Environment
Provide any relevant information about your setup, such as:
* Version of binance-connector-node 1.2.0
* Node version v16.7.0
* Operating system linux

**Comment by ishuen** [comment]
Thanks for reporting the issue. We will investigate it.

**Comment by ishuen** [comment]
The fix will be released in the next version.

**Comment by ishuen** [comment]
This issue should be fixed from v1.3.1.
